tracer_test.go

```diff
@@ -203,15 +204,15 @@ func TestTracerOptions(t *testing.T) {
 	openTracer, closer := NewTracer("DOOP", // respect the classics, man!
 		NewConstSampler(true),
 		NewNullReporter(),
-		TracerOptions.Logger(StdLogger),
+		TracerOptions.Logger(log.StdLogger),
```

Comment:
can we have a test that shows backwards compatibility by passing `jaeger.Logger` as `log.Logger`?